modules/core/shared/src/main/scala/gem/math/Angle.scala

```diff
@@ -245,6 +227,21 @@ object Angle {
 
 }
 
+trait AngleOptics { this: Angle.type =>
```

Comment:
Why do you require it to be mixed with `Angle`? Couldn't we mix the optics anywhere?

Reply:
It's intended as a mix-in but I guess in cases where you don't need access to the companion it could be unconstrained.